data/hai/unfettered 0 prologue.txt

```diff
@@ -297,57 +300,1253 @@ mission "Unfettered Jump Drive 3"
 			choice
-				`	(Yes.)`
 				`	(No.)`
 					defer
-			`	You grab a flashlight and open the hatch, and find a juvenile Hai there, with a large satchel slung over his shoulder. "You are the <first> <last>, yes?" he says. You nod. He says, "Is it true what is written here?"`
-			`	In his hands he is holding a piece of foil paper. You recognize it as the wrapper from one of the food rations that you have been transporting to the Unfettered, the "tribute" that they demand from the peaceful Hai. But having never unwrapped one of the rations, you had not realized that there is writing on the inside, a few words in the angular alphabet of the Hai. The Unfettered youth sees your look of confusion, and translates the message. "It says, 'Hai-home is your home too. Rejoin your people.'"`
+				`	(Yes.)`
+			`	You exit your ship and immediately the engineer starts leads you to the outfitter, "We will show you the weapon once you arrive. For now you will only be allowed to access one of them, but if you continue to prove of great assistance to us we may give you access to the others." They lead you trough the usual outfitter but then go to an area you never noticed before, probably because you were not meant to. You have to pass trough a check where they let you keep weapons but verify your identity and history.`
+			`	You are eventually taken to an underground zone, passing trough what looks like very basic weapon testing. Unfettered are there, doing small modifications and testing them, but never going really far in the tests. Seeing you look, the engineer explains. "We prefer to do most of our testing on the field of battle. Here we only make sure the weapon actually shoots before Unfettered captain can get their hands on it." Now that they mention it you notice a few captains looking at the tests, and one even requests a weapon to be taken to his ship, apparently satisfied with its performances.`
+			`	"Welcome to the special section of the outfitter, captain! It is only for our most distinguished guests. Now that you have sold us multiple jump drives, I can tell you that we can offer something way more valuable than money. For now, I can only offer you the mighty tripulse shredder, but eventually we will give you access to all kinds of weapons. 2 tripulse cannons for one jump drive and our sympathy is the deal I propose." They say, presenting the weapons to you. Now that you can take a closer look, you notice these weapons seem fairly similar to the rest of Hai weaponry, but enhanced someway or another. There even are some ion looking weapons, but one is smaller than an ion cannon whilst another one looks like a turreted version of it.`
```

Comment:
Just so uk this was in the tripulse shredder PR, but I'll rework it a bit for sure here

> So a few things here don't make a lot of sense.
> 
> 1. Testing in the vicinity of other expensive equipment stores feels like an inherently bad idea.

seems like smth the Unfettered could do, but I'll specify its well shielded

> 2. Underground anything is expensive and labour intensive to build, and also vulnerable to destruction. This feels like a very bad place for weapons testing of any kind when an open field or even a large warehouse would be options.

*nobody* can learn about this, secrecy is the major rule, also there being only one of these seems reasonable to me
the real testing happens in space indeed, they just make sure the weapon wont explode on your face

> 3. Presumably if captains are witnessing tests and taking them on, there must be some kind of system for iterating and classifying changes. This implies registration, and some kind of special trip that would be made. So none of this feels appropriate to what you would see going into a secure part of the outfitter.
>    I would expect instead that a selection of different "test" variants are on display for captains to take and report back on, maybe with videos or information about their distinctions, and there's the "most popular prototype" option with several in stock that might be on offer to the player.

alr, I added that the more they are appreciated, the more it will be developped (and thus produced)